src/Compilers/CSharp/Portable/Emitter/Model/PEModuleBuilder.cs

```diff
@@ -83,9 +83,14 @@ internal sealed override Cci.ICustomAttribute SynthesizeAttribute(WellKnownMembe
             return Compilation.TrySynthesizeAttribute(attributeConstructor);
         }
 
-        public sealed override IEnumerable<Cci.ICustomAttribute> GetSourceAssemblyAttributes()
+        public sealed override IEnumerable<Cci.ICustomAttribute> GetSourceAssemblyAttributes(bool isRefAssembly)
         {
-            return SourceModule.ContainingSourceAssembly.GetCustomAttributesToEmit(this.CompilationState, emittingAssemblyAttributesInNetModule: OutputKind.IsNetModule());
```

Comment:
>GetCustomAttributesToEmit [](start = 63, length = 25)

Consider adding an assert that the method that we used to call here is never called for an assembly symbol.